src/Cache/ContaoCacheWarmer.php

```diff
@@ -130,8 +139,12 @@ private function generateDcaCache($cacheDir)
         $dumper = new CombinedFileDumper($this->filesystem, new PhpFileLoader(), $cacheDir.'/contao', true);
         $processed = [];
 
-        /** @var SplFileInfo[] $files */
-        $files = $this->finder->findIn('dca')->files()->name('*.php');
+        try {
```

Comment:
We now have this block of 6 lines in two locations, this IMO justifies creation of a private method to prevent future errors.